sematic/log_reader.py

```diff
@@ -234,12 +257,63 @@ def _load_non_inline_logs(
     still_running: bool,
     cursor_file: Optional[str],
     cursor_line_index: int,
+    cursor_had_more_before: bool,
     max_lines: int,
     filter_strings: List[str],
 ) -> LogLineResult:
     """Load the lines for runs that are NOT inline"""
+    v1_prefix = v1_log_prefix(run_id, JobType.worker)
+    latest_v1_log_file = _get_latest_log_file(v1_prefix, cursor_file)
+    if latest_v1_log_file is not None:
+        return _load_non_inline_logs_v1(
+            run_id=run_id,
+            still_running=still_running,
+            cursor_file=cursor_file,
+            cursor_line_index=cursor_line_index,
+            cursor_had_more_before=cursor_had_more_before,
+            max_lines=max_lines,
+            filter_strings=filter_strings,
+        )
     prefix = log_prefix(run_id, JobType.worker)
```

Comment:
Please air the code and add comments for the 2 sections: v1 and v2.
Comment applies for inline as well.